docs/sources/writing-guide/documentation-structure/index.md

```diff
@@ -54,6 +54,26 @@ For more information about how to write tasks, refer to [Tasks]({{< relref "../t
 
 > **Note:** If a directory contains multiple pages or subdirectories, it is a branch bundle, and it must include an `_index.md` file. A directory containing only one page is a leaf bundle, and the content filename must be `index.md`.
 
-## Useful links
+## Pages and page bundles
 
-- For more information about branch bundles and leaf bundles, refer to [Page bundles](https://gohugo.io/content-management/page-bundles/).
+Each web page generated by Hugo comes from one of three source files:
+- `page/_index.md`: a Hugo branch bundle
```

Comment:
I think we need to map "branch" "leaf" and "page" to the terminology we're using in the rest of this file (Parent, Child, Task).

Reply:
Good suggestion, I did note that the rest of the file was using different terms. To me, "parent" and "child" are both relative terms. An intermediate branch bundle can be both a parent and a child, for instance `first/second/_index.md` in the following directory layout:
```
first/_index.md
first/second/_index.md
first/second/third.md
```

IIRC, the earlier file content talks about most branch bundles being intermediate pages and often being concepts. I guess that would mean that most "tasks" are leaf bundles or ordinary pages.

To be honest, I'm not sure I can marry the two sets of terminology cleanly because there are so many exceptions to the mapping.